definitions/contested/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -305,6 +312,13 @@
     "CaseTypeID": "FinancialRemedyContested",
     "CaseEventID": "FR_solicitorCreate",
```

Comment:
Should we remove caseworker access for the solicitor create event? If we are giving them their own unique one to create a case?

Reply:
I did, should be updated to have 'R' DRUD